index.src.html

```diff
@@ -635,25 +645,29 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   <h2 id="integration-with-css">Integration with CSS</h2>
 
   The CSS Standard does not specify how it fetches resources referenced from
-  stylesheets. However, implementations use the URL a given stylesheet was
-  loaded from as referrer for the requests for resources from that stylesheet.
-
-  Implementations should keep track of a referrer policy for each stylesheet
-  that should be used to create requests for resources from the respective
-  stylesheet.
-
-  For external stylesheets, the referrer policy should be
-  <a>"<code>no-referrer-when-downgrade</code>"</a> unless overwritten by an
-  `<code>Referrer-Policy</code>` header.
-
-  Note: If the stylesheet was loaded via a HTML <{link}> element with a
+  stylesheets. However, implementations should use the <a>location</a> of the
+  <a>stylesheet</a>, <a href="#strip-url">stripped for use as a referrer</a>,
```

Comment:
Nit: I don't think you need to strip the URL here. It gets stripped as part of https://w3c.github.io/webappsec-referrer-policy/#determine-requests-referrer (step 4-5).